server/sonar-server/src/main/resources/org/sonar/server/user/ws/current-example.json

```diff
@@ -15,11 +15,8 @@
   ],
   "permissions": {
     "global": [
-      "admin",
```

Comment:
why removing some permissions?
I guess the point of the example was also to show all available permissions (which I think is not a bad idea even if the WS is internal)